http-server/src/main/resources/openapi.yaml

```diff
@@ -3,92 +3,189 @@ info:
+          items:
+            $ref: '#/components/schemas/TransactionOp'
+        opts:
+          title: transaction opts
+          type: object
+          properties:
+            system-time:
+              schema:
+                $ref: '#/components/schemas/SystemTime'
+            default-tz:
+              schema:
+                $ref: '#/components/schemas/DefaultTz'
+            default-all-valid-time?:
+              type: boolean
+    Row:
+      type: object
+    Error:
+      type: object
+      properties:
+        xtdb.error/error-type:
+          type: string
+          example: "illegal-argument"
+        xtdb.error/error-key:
+          type: string
+          example: "xtql/malformed-table"
+        message:
+          type: string
+          example: "Illegal argument: ':xtql/malformed-table'"
+    Basis:
+      type: object
+      properties:
+        description: wall clock time for functions that require it (i.e. `(current-time)`)
+        current-time:
+          type: string
+          example: "2024-01-01T00:00"
+          format: date-time
+        tx:
+          description: specifies the latest transaction that’ll be visible to the query
+          schema:
+            $ref: '#/components/schemas/TransactionKey'
+        after-tx:
+          description: requires that the node has indexed at least the specified transaction
+          schema:
+            $ref: '#/components/schemas/TransactionKey'
+    Query:
+      type: object
+    QueryBody:
+      title: xtdb.server/query-body
+      required:
+        - query
+      type: object
+      properties:
+        query:
+          schema:
+            $ref: '#/components/schemas/Query'
+        basis:
+          schema:
+            $ref: '#/components/schemas/Basis'
+        basis-timeout:
+          description:  Timeout to wait for the basis
+          schema:
+            $ref: '#'#/components/schemas/Timeout'
+          nullable: true
+        args:
+          type: array
+          nullable: true
+          items:
+            type: object
+        default-all-valid-time?:
+          type: boolean
+        default-tz:
+          schema:
+            $ref: '#/components/schemas/DefaultTz'
 paths:
   /status:
     get:
       summary: Status
       description: Get status information from the node
       responses:
-        default:
-          description: ''
-          content: {}
+        '200':
+          description: Ok
+          content:
+            application/transit+json:
+              schema:
+                $ref: '#/components/schemas/Status'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Status'
   /tx:
     post:
       summary: Transaction
-      description: Submits a transaction to the cluster
+      description: > 
+        Submits a transaction to the cluster. 
+        See [XTQL Transactions](https://docs.xtdb.com/reference/main/xtql/txs) for details about different transaction operations.
```

Comment:
like this :ok-hand: same for queries please :)

these are also the endpoints for SQL txs/queries - could that link be included in the description too?